include/openssl/crypto.h

```diff
@@ -84,6 +84,7 @@ int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock);
 # define CRYPTO_MEM_CHECK_DISABLE 0x3   /* Control only */
 
 struct crypto_ex_data_st {
+    OPENSSL_CTX *ctx;
     STACK_OF(void) *sk;
 };
```

Comment:
Will people be very upset of we make `CRYPTO_EX_DATA` opaque?

Reply:
We're supposed to be trying not to break stuff wherever possible. I think this could be a case where we could break things so I think it is best to avoid it.

Reply:
Okie...